README.md

```diff
@@ -136,7 +136,7 @@ If you still need help to get started, feel free to reach out on our [community
 
 1. Install the development dependencies: `poetry install --with dev --with test`
 
-2. Install the pre-commit git hooks: `pre-commit install`
+2. Install the pre-commit git hooks: `pre-commit install && pre-commit install --hook-type commit-msg`
```

Comment:
This is coming from  https://commitizen-tools.github.io/commitizen/tutorials/auto_check/#method-1-add-git-hook-through-pre-commit

Reply:
@iiian , would pre-commit install not install it directly?

Reply:
Yeah! So, I agree this... looks goofy. and imprecise. And originally, this wasn't in my PR, because 💩. But, when I git cloned myself a second instance of `iiian/robyn` so that I could validate before making a PR, I tested setup used existing docs, w/ just `pre-commit install`. The result was that `git commit -m "definitely unconventional"` worked, **without** commitizen bonking me like I would have wanted. I double checked I was on a dev branch too.

I'm happy to dig deeper and see if I can achieve a solution that bootstraps devs to running commitizen checks with less!